exercises/practice/leap/.approaches/datetime-addition/content.md

```diff
@@ -0,0 +1,26 @@
+# `datetime` addition
+
+```python
+import datetime
+
+
+def leap_year(year):
```

Comment:
That is a great idea! I never bothered to find the "built-in" method for Python, but have done it elsewhere. I wonder if we can add that in a separate PR next week, as it is a substantial addition, and that way it can be reviewed thoroughly on its own? I'd be happy to work on something relatively small like that during the holiday week.

Reply:
I can do a separate PR for it at some point.

```python
from calendar import isleap

def leap_year(year):
    return isleap(year)
```

Feels pretty easy - although the discussion around it may or may not be.

Reply:
Thanks, I would do it now, but there is a need for updating the config,json, and introduction.md, and the performance content.md and snipped.md in addition to the section on the calendar approach itself. Plus, I have a backlog of two other of these waiting (Grains and Pangram) and I plan to do another one today for Isogram.

Reply:
Just so we don't conflict, I have ones in process for `word count`, `reverse string`, `acronym`, `raindrops`, and `space age`.  I am also going to do something on `robot simulator` when I get the energy.  

Eyeing `triangle`, `darts`, `etl`, and `matrix` as well ... but I won't overly-ambitious or greedy.